### ISSUE TITLE
Specified key was too long in initial migration

### ISSUE THREAD

**Issue by jessynt** [libern/someline-starter#1]
Hi,
I'm getting this error when I run migrate.

While installing on mysql 5.5.42
encoding utf8mb4 collation utf8mb4_unicode_ci

```
Migration table created successfully.


  [Illuminate\Database\QueryException]
  SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes (SQL: alter table `oauth_client_endpoints` add unique `oauth_cli
  ent_endpoints_client_id_redirect_uri_unique`(`client_id`, `redirect_uri`))



  [PDOException]
  SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes
```

**Comment by libern** [comment]
Hi, it is not working on version 5.7 and below.

#### Development Requirements

PHP: >=7.0
MySQL: >=5.7
Laravel 5: https://laravel.com/docs/5.2/installation
NodeJS: https://nodejs.org/
Bower: https://bower.io/
SQLite extention

**Comment by qqjt** [comment]
Same problem while using wamp:
mysql> select version();
+-----------+
| version() |
+-----------+
| 5.7.9     |
+-----------+
1 row in set

**Comment by libern** [comment]
@qqjt We recommend you try use homestead environment for better process. https://laravel.com/docs/5.2/homestead

**Comment by libern** [comment]
@qqjt also, Database should has Encoding utf8mb4 and Collation utf8mb4_unicode_ci

**Comment by qqjt** [comment]
I solved this problem by add max length for string columns:
https://github.com/qqjt/someline-starter/commit/2f6491ca33333320dbbc99d8945f8a8aa4b4b88e

For utf8mb4 encoding, the max length is 191 instead of 255, maybe that's why.

**Comment by libern** [comment]
@qqjt I was able to migrate without any modifications, you may reference to the Youtube guide to see the details.

**Comment by libern** [comment]
This issue has been fixed in v0.0.5.

**Comment by Muhammadinaam** [comment]
Hi, this is a nice package but I think that the developers have stopped working on it. Maybe you can try this package: https://github.com/Muhammadinaam/speed-admin. It supports Laravel 8+.
